main.py

```
@@ -1,609 +1,802 @@
+    # Gemini (lightweight generateContent)
+    if want("gemini"):
+        key = os.getenv("GEMINI_API_KEY") or get_credential("GEMINI_API_KEY")
+        if not key:
+            results["gemini"] = {"ok": False, "error": "missing key"}
+        else:
+            try:
+                url = f"https://generativelanguage.googleapis.com/v1beta/models/gemini-2.0-flash:generateContent?key={key}"
+                payload = {"contents": [{"parts": [{"text": "ping"}]}]}
+                r = requests.post(url, json=payload, timeout=10)
+                results["gemini"] = {"ok": r.status_code < 400, "status_code": r.status_code}
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)